services/motion/builtin/move_request.go

```diff
@@ -305,6 +301,21 @@ func (mr *moveRequest) obstaclesIntersectPlan(
 	return state.ExecuteResponse{}, nil
 }
 
+func (mr *moveRequest) createInputMap(baseExecutionState motionplan.ExecutionState) map[string][]referenceframe.Input {
+	inputMap := referenceframe.StartPositions(mr.localizaingFS)
```

Comment:
This should call CurrentInputs() on everything InputEnabled, not assume their inputs are zero.

Otherwise we're laying landmines for ourselves to hit when we start allowing arms on bases